frontend/package.json

```diff
@@ -53,10 +55,8 @@
     "pinia": "^2.1.6",
     "qrcode": "^1.5.3",
     "qrcode.vue": "^3.4.1",
-    "reduced-motion": "^1.0.2",
+    "reduced-motion": "^1.0.4",
```

Comment:
Nice 😉

Reply:
I noticed a warning between the time I uninstalled tailwindcss and installed it as a dev dep. that reduced-motion needed that dependency, I initially wanted to add tailwind support and added it in peerDependencies but later realised that it didn't need anything to work with tailwind. This was the reason for the update there